src/serialize_sequence.cc

```diff
@@ -146,7 +146,7 @@ std::ostream& operator<<(std::ostream& stream, const Sequence& sequence)
     return stream;
 }
 
-void store_step_setup_script(const std::filesystem::path& lua_file, const Sequence& seq)
+void store_sequence_parameter(const std::filesystem::path& lua_file, const Sequence& seq)
```

Comment:
Renaming is a good idea! I'd propose store_sequence_parameter**s**, though. There is more than one.